lib/src/base/utils.dart

```diff
@@ -198,3 +198,25 @@ enum UpdateFrequency {
   /// Reports duration at every 200 milliseconds.
   low
 }
+
+/// This enum is meant to be used as a parameter of setReleaseMode method.
+///
+/// It represents the behavior of AudioPlayer when an audio is finished or stopped.
+enum ReleaseMode{
```

Comment:
Let's not change the name of the enum because then it will be a bigger breaking change